src/main/scala/gestalt/Symbols.scala

```diff
@@ -15,5 +15,8 @@ trait Symbols { this: Toolbox =>
 
     /** type of a member with respect to a prefix */
     def asSeenFrom(mem: Symbol, prefix: Type): Type
+
+    /** subst symbols in tree */
+    def subst(tree: tpd.Tree)(from: List[Symbol], to: List[Symbol]): tpd.Tree
```

Comment:
Please let me know when the PR is ready for review. I saw recent progress, but I'm not sure whether the current state of code is final.